pkgs/development/tools/misc/roswell/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, autoconf, automake, curl }:
+
+stdenv.mkDerivation rec {
+  name = "roswell-${version}";
+  version = "17.9.10.82";
+
+  src = fetchgit {
+    url = "https://github.com/roswell/roswell";
+    sha256 = "1064lj5hy70j7nkf46h7pg5yvksgaj3k9aha7gp09hd826857q4m";
+    rev = "af18d986a33502156242c908f1f412a170f64721";
+  };
+  
+  preConfigure = ''
+    sh bootstrap
+  '';
+
+  nativeBuildInputs = [ autoconf automake curl ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/roswell/roswell;
+    description = "lisp installer and launcher for major environments that just works";
```

Comment:
Capitalize the first letter of the description, https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.